Gruntfile.js

```diff
@@ -1,5 +1,39 @@
 'use strict';
 
+function proxy() {
+  var request = require('request');
+
+  return function proxyMiddleware(req, res, next) {
+    if (req.method.toUpperCase() === 'OPTIONS') {
```

Comment:
Please remember to filter out the `cookies` header, you do not want to forward those.